main.py

```diff
@@ -0,0 +1,8 @@
+from neuraxle.distributed import RestWorker
+
+
+if __name__ == '__main__':
+    worker = RestWorker(port=5000)
+    worker.start(upload_folder='/home/alexandre/Documents/cluster')
```

Comment:
Please read this: https://flask.palletsprojects.com/en/1.1.x/tutorial/deploy/
This means we might need another pip dependency. OR otherwise we don't provide a main, and just document this. 

I would rather just document this maybe. I'm not sure.

Reply:
dead code for testing purposes